src/main/webapp/js/pages/login.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Gets user information and populates the topbar with their image and name
+ * @param {GoogleUser} googleUser the currently logged in user
```

Comment:
wasn't sure whether to put Object or the type from the Google JavaScript client reference
https://developers.google.com/identity/sign-in/web/reference#googleauthcurrentuserget

Reply:
Personally I put the specific object type from the library I used and that was fine.